docs/docs/guides/evaluation/scorers.md

```diff
@@ -0,0 +1,318 @@
+
+# create a scorer with a column mapping the `text` parameter to the `news_article` data column
+scorer = SummarizationScorer(column_map = {"text" : "news_article"})
+```
+Here, the `text` parameter in the score method will receive data from the `news_article` column.
+
+
+## Predefined Scorers
+
+**LLM-evaluators**
+
+The pre-defined scorers that use LLMs support the OpenAI, Anthropic, Google GenerativeAI and MistralAI clients. They also uses weave's `InstructorLLMScorer` class, so you'll need to install the [`instructor`](https://github.com/instructor-ai/instructor) Python package to be able to use them.
+
+### `HallucinationFreeScorer`
+
+This scorer checks if your AI system's output includes any hallucinations based on the input data.
+
+```python
+from weave.scorers import HallucinationFreeScorer
+
+llm_client = # initialize your LLM client here
+
+scorer = HallucinationFreeScorer(
+    client=llm_client, 
+    model_id="gpt4o"
+)
+```
+
+**Customization:**
+- Customize the `system_prompt` and `user_prompt` attributes of the scorer to define what "hallucination" means for you.
+
+**Notes:**
+- The `score` method expects an input column named `context`. If your dataset uses a different name, use the `column_map` attribute to map `context` to the dataset column.
+
+---
+
+### `SummarizationScorer`
+
+Use an LLM to compare a summary to the original text and evaluate the quality of the summary.
+
+```python
+from weave.scorers import SummarizationScorer
+
+llm_client = # initialize your LLM client here
+
+scorer = SummarizationScorer(
+    client=llm_client, 
+    model_id="gpt4o"
+)
+```
+
+**How It Works:**
+
+This scorer evaluates summaries in two ways:
+
+1. **Entity Density:** Checks the ratio of unique entities (like names, places, or things) mentioned in the summary to the total word count in the summary in order to estimate the "information density" of the summary. Uses an LLM to extract the entities. Similar to how entity density is used in the Chain of Density paper, https://arxiv.org/abs/2309.04269
+
+2. **Quality Grading:** Uses an LLM-evaluator to grade the summary as `poor`, `ok`, or `excellent`. These grades are converted to scores (0.0 for poor, 0.5 for ok, and 1.0 for excellent) so you can calculate averages.
+
+**Customization:**
+- Adjust `summarization_evaluation_system_prompt` and `summarization_evaluation_prompt` to define what makes a good summary.
+
+**Notes:**
+- This scorer uses the `InstructorLLMScorer` class.
+- The `score` method expects the original text that was summarized to be present in the `input` column of the dataset. Use the `column_map` class attribute to map `input` to the correct dataset column if needed.
+
+
+---
+
+### `OpenAIModerationScorer`
+
+The `OpenAIModerationScorer` uses OpenAI's Moderation API to check if the AI system's output contains disallowed content, such as hate speech or explicit material.
+
+```python
+from weave.flow.scorers.moderation_scorer import OpenAIModerationScorer
```

Comment:
The flow import here feels awkward -- maybe better to just do `weave.scorers.moderation_scorer`